pkg/ephemeral/spdz.go

```diff
@@ -198,7 +213,23 @@ func (s *SPDZEngine) Activate(ctx *CtxConfig) ([]byte, error) {
 	return nil, errors.New("no MPC parameters specified")
 }
 
-// Compile compiles a SPDZ application.
+func (s *SPDZEngine) getNumberOfThreads() (int, error) {
+	file, err := os.Open(defaultSchedulePath)
```

Comment:
Should this be `s.schedulePath` instead of the constant value?

Reply:
definitely